gcp/appengine/index.yaml

```diff
@@ -80,6 +80,15 @@ indexes:
       direction: desc
 
   # ecosystem + search_indices queries is served by merging the two indexes above
+  
+  # To allow sitemap generation to project and sort on last_modified
+  - kind: Bug
```

Comment:
This index already exists on prod, but was not recorded here (and doesn't exist on test). I think I removed it when I did some cleanup and when we switched the frontend from sorting by last modified to published. 

I have now added this on test as well.